packages/react-scripts/scripts/audit.js

```diff
@@ -0,0 +1,75 @@
+// @remove-file-on-eject
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+'use strict';
+
+const { createServer } = require('http');
+const { writeFileSync } = require('fs');
+const { join } = require('path');
+const { choosePort } = require('react-dev-utils/WebpackDevServerUtils');
+const open = require('open');
+const handler = require('serve-handler');
+const lighthouse = require('lighthouse');
```

Comment:
Running chrome in headless would be amazing! I could see some cool CI use cases to fail when below a threshold. Potentially using puppeteer would make that trivial for us, removes the need to have chrome installed to use this functionality, and also provide the option to run headed if needed for debugging etc.

Reply:
https://github.com/justinribeiro/lighthouse-jest-example

Reply:
Yes, so there's an option for headless. I left it "headfull" as the default intentionally - I feel that for some people, it's nice to see how it works - I guess we can rely on a flag or config for opting out of that?

What do you think @ianschmitz?

Reply:
On second thought, including `puppeteer` as a direct dependency would greatly increase the `node_modules` size which isn't great:

`When you install Puppeteer, it downloads a recent version of Chromium (~170MB Mac, ~282MB Linux, ~280MB Win)`.

Documentation could likely handle this:

`If you want to run this in CI, make sure Chrome/Chromium is installed...` (or use puppeteer etc.)

Reply:
Yes, and I think we should also handle this in the script... I'm sure we can.